src/components/form/form.jsx

```diff
@@ -85,17 +95,19 @@ const Form = React.forwardRef((props, forwardedRef) => {
 });
 
 Form.propTypes = {
-  submitText: PropTypes.string,
+  autoSave: PropTypes.bool,
   children: PropTypes.func.isRequired,
   onChange: PropTypes.func,
   onSubmit: PropTypes.func,
   readOnly: PropTypes.bool,
-  refs: PropTypes.arrayOf(
-    PropTypes.shape({ current: PropTypes.any }).isRequired,
-  ).isRequired,
+  refs: PropTypes.arrayOf((
```

Comment:
Why do we need the extra parens here?

Reply:
It's a way to get around the "trailing comma" lint for function parameters.

e.g.

```js
someFunction(
  param1,
);
```

vs

```js
someFunction((
  param1
));
```

This is usually pretty annoying with arrow functions:

```jsx
someArray.map(element => 
  <div>{element}</div>,
)
```

vs

```js
someArray.map(element => (
  <div>{element}</div>
))
```